src/components/item-display-panel/item-display-panel.component.ts

```diff
@@ -1,18 +1,24 @@
-import { Component, Directive, Input, SimpleChange, Output, EventEmitter } from '@angular/core';
+import { Component, Directive, Input, SimpleChange, Output, EventEmitter, Renderer2, ElementRef, Host } from '@angular/core';
 
 @Component({
     selector: 'ux-item-display-panel',
     templateUrl: './item-display-panel.component.html',
     host: {
         '(document:click)': 'clickOff($event)',
-        '(document:keyup.escape)': 'visible = false'
+        '(document:keyup.escape)': 'visible = false',
+        '[class.inlineHost]' : 'inline', 
```

Comment:
Use kebab case rather than camel case in class names. `inline` and `visible` might even be enough for the class names